sphinx/util/docutils.py

```diff
@@ -464,6 +467,42 @@ def __init__(self, document: nodes.document, builder: "Builder") -> None:
         self.config = builder.config
         self.settings = document.settings
 
+    def dispatch_visit(self, node):
+        """
+        Dispatch node to appropriate visitor method.
+        The priority of visitor method is:
+
+        1. ``self.visit_{node_class}()``
+        2. ``self.visit_{supre_node_class}()``
+        3. ``self.unknown_visit()``
```

Comment:
If user add a `visit_node` method, user won't reach `unknown_visit`.
However I don't think there is a real problem.